grader_host/tools/syncECR.js

```diff
@@ -88,7 +88,7 @@ async.series(
       var sql = `
         SELECT DISTINCT external_grading_image
         FROM questions AS q
-        JOIN pl_courses AS plc ON (q.course_id = plc.id)
+        JOIN pl_courses AS plc ON (q.course_id = plc.id) -- does it matter if I change this?
```

Comment:
This whole file has since been deleted; this should just go away when you rebase.